### ISSUE TITLE
Minor Enhancements for better training

### ISSUE THREAD

**Issue by 23Tarandeep57** [parassoni04/Calculator#22]
1- Fix the random seed for reproducibility
2- Increase learning rate even with 0.1, cost is not exploding with sigmoid (atleast for 20 epochs, need numpy acceleration if want to train properly)
3- Experiment with different activation functions(Like ReLU), sigmoid can be prone to gradient vanishing leading to slow convergence
4- Try He Initialization with ReLU so Even increased learning rate gradients won't explode

**Comment by DracoRooks** [comment]
Noted, will look into 'em. Thanx!
